src/chunkserver/copyset_node.cpp

```diff
@@ -168,6 +172,11 @@ int CopysetNode::Init(const CopysetNodeOptions &options) {
     raftNode_ = std::make_shared<RaftNode>(groupId, peerId_);
     concurrentapply_ = options.concurrentapply;
 
+    // init braft lease
+    if (options.enbale_lease_read) {
+        braft::FLAGS_raft_enable_leader_lease = true;
```

Comment:
we will have lots of copyset node in one chunkserver, so it's not a good idea to enable/disable lease read multiple times.

you can read the configuration item from file and enable/disable in Chunkserver::Run https://github.com/opencurve/curve/blob/7e8e774e6425051481aa838aefe866b13c51061f/src/chunkserver/chunkserver.cpp#L84